CondFormats/CTPPSReadoutObjects/plugins/TotemDAQMappingESSourceXML.cc

```diff
@@ -233,23 +234,24 @@ void TotemDAQMappingESSourceXML::setIntervalFor(const edm::eventsetup::EventSetu
   {
     const auto &bl = configuration[idx];
 
+    edm::EventRange range = bl.validityRange;
+
     // event id "1:min" has a special meaning and is translated to a truly minimal event id (1:0:0)
-    EventID startEventID = bl.validityRange.startEventID();
-    if (startEventID.event() == 1)
-      startEventID = EventID(startEventID.run(), startEventID.luminosityBlock(), 0);
+    if (range.startEventID()==edm::EventID(1, 0, 1))
+      range = edm::EventRange(edm::EventID(1, 0, 0), range.endEventID());
```

Comment:
just for me to understand: using event number '0' has what exact special meaning?
or phrased differently, what kind of magic happens in `EventRange` and `EventID` that prevents the kind of failure mentioned by @franzoni from happening again?